workflow/admin.py

```diff
@@ -87,6 +87,13 @@ def formfield_for_foreignkey(self, db_field, request=None, **kwargs):
 
 class GaitIDInlineAdmin(admin.StackedInline):
     model = GaitID
+    fields = ['gaitid', 'donor', 'donor_dept']
+    readonly_fields = ['fund_codes',]
+    extra = 0
```

Comment:
On the Program page showing the `GaitIDInlineAdmin`, I don't see the fund_codes listed. I think the `fields` list needs `fund_codes` added.

Reply:
Good catch Blake! Unfortunately I don’t think we can put an inline inside an inline 😭